components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -39,15 +41,16 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
     this.layoutFacade.showSidebar(Sidebar.Compliance);
     this.reportLoading = true;
     this.layoutFacade.ShowPageLoading(true);
+    this.returnParams = this.reportQueryService.formatReturnParams();
```

Comment:
onInit - we call the service right away so that we can populate the breadcrumb with the return value immediately.